pkgs/development/python-modules/pydicom/default.nix

```diff
@@ -9,21 +9,49 @@
 , pillow
 }:
 
-buildPythonPackage rec {
-  version = "2.1.1";
+let
   pname = "pydicom";
```

Comment:
Can we move this back into buildPythonPackage?

Reply:
I did this way to reuse pname in `test_data` fetching.